tests/test_lambda_handler.py

```diff
@@ -79,6 +86,12 @@ def test_syslog_3_extract_time():
     assert resp == 869077200
 
 
+def test_syslog_4_extract_time():
+    message = "syslog INFO [2019/May/20:04:01:30 PM] temp"
+    resp = fluentdhec.lambda_function.extract_time(message)
+    assert resp == 1558364490
+
+
```

Comment:
Little confused as to why there's two functions with the same name here?

Reply:
Good catch, they are no longer the same name :D